packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -192,14 +192,18 @@ export default class MeetingInfoV2 {
 
     const options = {
       method: HTTP_VERBS.POST,
-      service: 'webex-appapi-service',
-      resource: 'meetingInfo',
       body
     };
 
     const directURI = await MeetingInfoUtil.getDirectMeetingInfoURI(destinationType);
 
-    if (directURI) options.directURI = directURI;
+    if (directURI) {
+      options.uri = directURI;
+    }
+    else {
+      options.service = 'webex-appapi-service';
+      options.resource = 'meetingInfo';
```

Comment:
is this the same value as `MEETINGINFO` in `packages/node_modules/@webex/plugin-meetings/src/constants.ts`? If yes then please use the constant.

Reply:
this represents part of a url. A resource could be meetingInfo/something/something else. I don't think we want to use constants for this. It is a coincidence that it is the same as the one used for the locus meeting info